### ISSUE TITLE
Command line build failed when installer image mounted

### ISSUE THREAD

**Issue by fuzzylogiq** [MagerValp/AutoDMG#176]
Had this slightly impenetrable traceback when running command line build while I still had the BaseSystem dmg mounted (possibly as a different user to the one running AutoDMG):
```
Traceback (most recent call last):
  File "/Applications/AutoDMG.app/Contents/Resources/IEDWorkflow.py", line 89, in handleDetachResult_
    self.delegate.detachFailed_details_(result[u"dmg-path"], result[u"error-message"])
AttributeError: 'IEDCLIController' object has no attribute 'detachFailed_details_'
2017-03-27 14:09:47.812 AutoDMG[40278:971248] AutoDMG died with an uncaught exception 'IEDCLIController' object has no attribute 'detachFailed_details_': Traceback (most recent call last):
  File "/Applications/AutoDMG.app/Contents/Resources/main.py", line 215, in main
    status = cli_main(decoded_argv)
  File "/Applications/AutoDMG.app/Contents/Resources/main.py", line 169, in cli_main
    return args.func(args)
  File "/Applications/AutoDMG.app/Contents/Resources/IEDCLIController.py", line 174, in cmdBuild_
    self.waitBusy()
  File "/Applications/AutoDMG.app/Contents/Resources/IEDCLIController.py", line 62, in waitBusy
    nextfire = runLoop.limitDateForMode_(NSDefaultRunLoopMode)
  File "/Applications/AutoDMG.app/Contents/Resources/IEDWorkflow.py", line 89, in handleDetachResult_
    self.delegate.detachFailed_details_(result[u"dmg-path"], result[u"error-message"])
AttributeError: 'IEDCLIController' object has no attribute 'detachFailed_details_'
```

If I detach the image the error goes away, natch, but I wonder if this could be caught a bit more gracefully?

AutoDMG version: 1.7.3
Host OS: 10.11.6 
Installer source: 10.11.6 15G31

**Comment by fuzzylogiq** [comment]
Hm...I *may* have also moved the original BaseSystem.dmg (just noticing that release notes guard around InstallESD.dmg moving)

**Comment by MagerValp** [comment]
This was fixed in a43f5514b091fb14b7c14ea7db91971917f9b723 and will be in the next release.
